setup.cfg

```diff
@@ -0,0 +1,41 @@
+[metadata]
+name =  flask-container-scaffold
+version = 0.0.1
+author = Jason Guiditta
+author_email = jason.guiditta@gmail.com
+description = Configuration layer to aid in deployment of Flask apps
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/release-depot/flask-container-scaffold
+project_urls =
+    Bug Tracker = https://github.com/release-depot/flask-container-scaffold/issues
+license = MIT
+license_files = LICENSE
+classifiers =
+    Development Status :: 3 - Alpha
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = src
+packages = find:
+python_requires = >=3.6
+install_requires =
+    flask
+    toolchest
```

Comment:
Should toolchest be a default? I don't think we've used it in many of the other services, so far? Or do we plan on that changing?

Reply:
Yes, the new code uses toolchest, but is coming in the next patch